src/BTR3.ohm

```diff
@@ -120,10 +122,17 @@ BTR3 {
     hh = ("0".."1" digit) | ("2" "0" .. "3")
     mm = "0".."5" digit
 
-    delim = ","
+    newLine = ("\r\n" | "\n" | "\r")
+    delim =
+        | "," -- comma_delimiter
+        | "/" newLine "88," -- continuation_record_delimiter
+
     eor = "/"
 
-    alphaNumeric    (BTR3 Alphanumeric - string not containing comma or slash) 
-                    = ~(delim | eor) any
+    alphaNumeric (BTR3 Alphanumeric - string not containing comma or slash) 
+                 = ~("," | "/") any
+
+    continuationWithinTextField = newLine "88,"
```

Comment:
However, a text field does not have a delimiter at the end of a logical record. we want to parse
```
...text
88,andmoretext
```
as `textandmoretext`

![Screen Shot 2021-03-02 at 4 11 46 PM](https://user-images.githubusercontent.com/9813258/109715916-36c06a00-7b72-11eb-8ed4-b1b43762b9a4.png)